cmd/bootkube/recover.go

```diff
@@ -66,6 +70,22 @@ func runCmdRecover(cmd *cobra.Command, args []string) error {
 			return err
 		}
 		backend = recovery.NewEtcdBackend(etcdClient, recoverOpts.etcdPrefix)
+	case recoverOpts.etcdBackupFile != "":
```

Comment:
thought about it. but the logging will be duplicated. it will look like

```
Attempting recovery using etcd backup file at ...
Attempting recovery using etcd cluster at ...
```

but, yea, i will try to clean up code duplication here in another way.